handout/modules/shared-di-tree.md

```diff
@@ -39,24 +39,8 @@ import { SharedModule } from './shared/shared.module';
 export class AppModule {}
 ```
 
-In contrast, when import the same module in our `LazyModule` we will not call the `forRoot` method because we don't want to register the service again in a different level of the DI tree, so the declaration of the `LazyModule` doesn't change.
+Since we import the module with `forRoot` in our root application module, we **do not** need to import it in `LazyModule`. Reimporting core root modules defined with `forRoot` in lazily loaded modules can cause issues in your application.
```

Comment:
Yep so firstly the docs mention that calling `forRoot` in another module, particularly lazy loaded modules, can trigger a run time error. However, if `SharedModule` provides a service to a lazy loaded module (through `providers` for example), the lazy loaded module would make it's own copy of the service.

Could definitely be more clear when I cover it in this section.